api/index.js

```diff
@@ -215,6 +234,71 @@ app.get('/api/v1/batsignal/council-events', async (req, res) => {
   }
 });
 
+/**
+ * Add EDP data to database
+ * @param {key, values} key and values
+ */
+app.post('/api/v1/edp', validateToken, async (req, res) => {
+  try {
+    const { key, values } = req.body;
+    const client = await getClient();
+    const query = `
+    INSERT INTO edp_metric (
```

Comment:
Apply the same formatting for the query as above.

Reply:
added